Sources/Datadog/RUM/RUMMonitor/Scopes/RUMApplicationScope.swift

```diff
@@ -34,6 +48,24 @@ internal class RUMApplicationScope: RUMScope {
     let context: RUMContext
 
     func process(command: RUMCommand) -> Bool {
+        if let currentSession = sessionScope {
+            let shouldRefreshSession = currentSession.process(command: command)
+            if shouldRefreshSession {
+                let refreshedSession = RUMSessionScope(parent: self, dependencies: dependencies)
+                sessionScope = refreshedSession
+                _ = refreshedSession.process(command: command)
+            }
+        } else {
+            switch command {
+            case .startView:
+                let newSession = RUMSessionScope(parent: self, dependencies: dependencies)
+                sessionScope = newSession
+                _ = newSession.process(command: command)
+            default:
+                break
+            }
```

Comment:
i didn't understand the logic here: if a session is expired, new one can start with any command; however if there is no expired session, new session can only start with `startView`. 
why are those 2 cases different?